exchanges/kraken/futures_types.go

```diff
@@ -378,7 +378,7 @@ type FuturesFillsData struct {
 	Fills []struct {
```

Comment:
Out of scope/separate issue: we should rid anon structs because it can be hard to mock externally if required